sqltest/testdata/show_schemas_test_script.txt

```diff
@@ -0,0 +1,7 @@
+use test;
+
+show schemas;
+
+use test0;
+
+show schemas;
```

Comment:
I think it would be good to extend the test a bit more:

```
use test;

show schemas;

use test0;

show schemas;

-- now use another schema - test0 should disappear as it is empty;
use test1;
show schemas;

-- now create a source in this schema;
create source foo (...);

-- now go to another schema - test1 should still exist as not empty;
use test2;
show schemas;

-- now go back to test1 and drop the source, then back to test2 - test0 should have disappeared;
use test1;
drop source foo;

use test2;
show schemas;
```

Reply:
Hi @naveen246 if you're stuck on this, happy to help out if you like?